stix_shifter_modules/sentinelone/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,270 @@
+      ],
+      "src_ref.value": [
+        "srcIp"
+      ],
+      "dst_ref.value": [
+        "dstIp"
+      ]
+    }
+  },
+  "domain-name": {
+    "fields": {
+      "value": [
+        "dnsRequest",
+        "dnsResponse"
+      ]
+    }
+  },
+  "url": {
+    "fields": {
+      "value": [
+        "url"
+      ]
+    }
+  },
+  "process": {
+    "fields": {
+      "command_line": [
+        "srcProcCmdLine",
+        "tgtProcCmdLine"
+      ],
+      "created": [
+        "srcProcStartTime",
+        "tgtProcStartTime"
+      ],
+      "pid": [
+        "srcProcPid",
+        "tgtProcPid",
+        "srcProcParentPid"
+      ],
+      "name": [
+        "srcProcName",
+        "srcProcParentName",
+        "tgtProcName"
+      ],
+      "parent_ref.name": [
+        "srcProcParentName"
+      ],
+      "creator_user_ref.user_id": [
+        "srcProcUser",
+        "tgtProcUser"
+      ],
+      "parent_ref.pid": [
+        "srcProcParentPid"
+      ]
+    }
+  },
+  "file": {
+    "fields": {
+      "name": [
+        "fileFullName"
+      ],
+      "size": [
+        "tgtFileSize"
+      ],
+      "hashes.MD5": [
+        "tgtFileMd5",
+        "tgtFileOldMd5"
+      ],
+      "hashes.'SHA-1'": [
+        "tgtFileSha1",
+        "tgtFileOldSha1"
+      ],
+      "hashes.'SHA-256'": [
+        "tgtFileSha256",
+        "tgtFileOldSha256"
+      ],
+      "parent_directory_ref.path": [
+        "tgtFilePath",
+        "tgtFileOldPath"
+      ],
+      "created": [
+        "tgtFileCreatedAt"
+      ],
+      "modified": [
+        "tgtFileModifiedAt"
+      ]
+    }
+  },
+  "directory": {
+    "fields": {
+      "path": [
+        "tgtFilePath",
+        "tgtFileOldPath"
+      ]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": [
+        "loginAccountSid",
+        "srcProcUser",
+        "tgtProcUser"
+      ],
+      "account_login": [
+        "loginsUserName"
+      ],
+      "is_privileged": [
+        "loginIsAdministratorEquivalent"
+      ],
+      "display_name": [
+        "loginAccountName"
+      ]
+    }
+  },
+  "windows-registry-key": {
+    "fields": {
+      "key": [
+        "registryKeyPath"
+      ],
+      "values[*]": [
+        "registryValue"
+      ]
+    }
+  },
+  "x-sentinelone-process": {
+    "fields": {
+      "image_path": [
+        "srcProcImagePath",
+        "tgtProcImagePath"
+      ],
+      "integrity_level": [
+        "srcProcIntegrityLevel",
+        "tgtProcIntegrityLevel"
+      ],
+      "publisher": [
+        "srcProcPublisher",
+        "tgtProcPublisher"
+      ],
+      "parent_process_start_time": [
+        "srcProcParentStartTime"
+      ],
+      "parent_image_path": [
+        "srcProcParentImagePath"
+      ]
+    }
+  },
+  "x-sentinelone-network-action": {
+    "fields": {
+      "connection_status": [
+        "netConnStatus"
+      ]
+    }
+  },
+  "x-sentinelone-login": {
+    "fields": {
+      "machine_ip": [
+        "srcMachineIP"
```

Comment:
Is this is an ip address, it should be searchable under `ipv4-addr:value`.

Reply:
The changes has been done in From_stix and To_Stix map file.

Reply:
I see you removed `machine_ip` from the mappings. isn't it support by the api anymore?

Reply:
Field "srcMachineIP" is now mapped with the IPV4-addr and IPV6-addr, hence removed it from the custom extension object.